### PR TITLE
added: hook up RPTRST CONV output to restart file if requested

### PR DESCRIPTION
**Pull request by akva2**
Sits on top of https://github.com/OPM/opm-simulators/pull/5053

Downstream of https://github.com/OPM/opm-common/pull/3838

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by akva2** [comment]
MOD4_XXX.DATA has CONV entries. either reference data or case needs to be updated.

**Comment by bska** [comment]
> MOD4_XXX.DATA has CONV entries. either reference data or case needs to be updated.

I think probably the test cases in this instance.  While, ideally, we'll have deterministic residual convergence history as new development occurs, it is nevertheless conceivable that there will be slight changes which will show up as differences in `CNV_*` without affecting the converged numerical solution.

**Comment by akva2** [comment]
yeah, i was thinking the same.

**Comment by akva2** [comment]
jenkins build this opm-tests=1116 please

**Comment by akva2** [comment]
jenkins build this please

**Comment by bska** [comment]
jenkins build this please

**Review by bska — COMMENTED** [review]
Overall this looks good to me.  The only thing I'm not quite sure about is the location where we identify the worst cells.  By excluding `iteration == 0` we **might** end up giving a misleading impression.

File: opm/simulators/flow/BlackoilModelEbos.hpp [hunk 1]
```diff
@@ -373,6 +392,8 @@ namespace Opm {
                 dx_old_ = 0.0;
                 convergence_reports_.push_back({timer.reportStepNum(), timer.currentStepNum(), {}});
                 convergence_reports_.back().report.reserve(11);
+            } else {
+                rst_conv_.update(ebosSimulator_.model().linearizer().residual());
```
[the remark this answers is not in the document]
Comment (akva2) [reply opm/simulators/flow/BlackoilModelEbos.hpp:396]: it's actually identifying the cells for ```iteration == 0```, but it lacks for last iteration before convergence is achieved. so i can move it.

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  This looks good to me now and I'll merge into master.

**Pull request merged**